bounties/npm/npm-programmatic/1/vulnerability.json

```diff
@@ -0,0 +1,50 @@
+{
+	"PackageVulnerabilityID": 1,
+	"DisclosureDate": "2020-08-17",
+	"AffectedVersionRange": "*",
+	"Summary": "Command Injection",
+	"Contributor": {
+		"Discloser": "",
+		"Fixer": ""
+	},
+	"Package": {
+		"Registry": "npm",
+		"Name": "npm-programmatic",
+		"URL": "https://www.npmjs.com/package/npm-programmatic",
+		"Downloads": "629189"
+	},
+	"CWEs": [
+		{
+			"ID": "CWE-79",
+			"Description": ""
+		}
+	],
+	"CVSS": {
+		"Version": "3.1",
+		"AV": "L",
+		"AC": "H",
+		"PR": "L",
+		"UI": "N",
+		"S": "U",
+		"C": "L",
+		"I": "L",
+		"A": "N",
+		"E": "",
+		"RL": "",
+		"RC": "",
+		"Score": "3.6"
```

Comment:
Higher the `score` since it's a `arbitrary command execution`

Reply:
Thanks for pointing this out @Mik317, do you have a score in mind that would suit this vulnerability better?

Reply:
Hey @benharvie :smile:, the similar issues on the platform have been considered around `7.5` and `8.0` basing on the possibility `affected method/function` could be used in a real-scenario.

Basing on this I would say `7.5` is ok :smile:

Cheers,
Mik

Reply:
Updated as recommended, cheers! ❤️